src/main/java/com/shipmentEvents/demo/BufferedReaderExample.java

```diff
@@ -0,0 +1,16 @@
+package com.company;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+public class BufferedReaderExample {
+    public static void main(String[] args) {
+
+
+            try {
+                BufferedReader br = new BufferedReader(new FileReader("test.txt"));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *br*. Execution paths that do not contain closure statements were detected. Ensure *br* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).